src/appkit.cs

```diff
@@ -16396,6 +16787,26 @@ public partial interface NSTextField : NSAccessibilityNavigableStaticText {
 		[Mac (10,11)]
 		[Export ("allowsDefaultTighteningForTruncation")]
 		bool AllowsDefaultTighteningForTruncation { get; set; }
+
+		[Mac (10,12)]
+		[Static]
+		[Export ("labelWithString:")]
+		NSTextField LabelWithString (string stringValue);
```

Comment:
`WithString` has not value (parameter makes it clear) and we do not know if we get (e.g. reuse) one or create (brand new) label. Maybe `CreateLabel` ? depending on context